src/yunohost/app.py

```diff
@@ -1303,8 +1309,7 @@ def app_checkurl(auth, url, app=None):
     path = url[url.index('/'):]
     installed = False
 
-    if path[-1:] != '/':
-        path = path + '/'
+    domain, path = _normalize_domain_path(domain, path)
```

Comment:
`_normalize_domain_path` doesn't had a trailing "/", this code replacement is not equivalent.

Reply:
This is to keep the same result than the `register_url`. Actually the `register_url` function use this function. I think we need to be coherent.